content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md

```diff
@@ -0,0 +1,153 @@
+
+![](/img/code-repository.png "Figure 7.  Source code for models is stored on Github in a private repository. ")
+
+**Model Deployment**
+
+The next step is to deploy the best model chosen from the MLflow experiment. Here we are using KServe which is a standard Model Inference Platform on Kubernetes, built for highly scalable use cases.
+
+Kube Flow Serving, also known as KServe, is a component of the Kube Flow ecosystem and is designed to serve machine learning models in Kubernetes environments. Kube Flow itself is an open-source platform for deploying, monitoring, and managing machine learning models on Kubernetes.
+
+KServe provides several features and benefits:
+
+1. Scalability: It is designed to handle serving machine learning models at scale. You can easily scale up or down based on the traffic and load requirements.
+2. Multi-Framework Support: KServe supports serving models trained in various machine learning frameworks, such as Tensor Flow, PyTorch, scikit-learn, and others.
+3. Advanced Deployment Strategies: KServe supports various deployment strategies, for example  Canary deployments or Blue-Green deployments,  allowing you to roll out new model versions gradually and monitor their performance.
+4. Monitoring and Metrics: It provides metrics and monitoring capabilities, allowing you to keep track of model performance, errors, and other relevant statistics.
+5. Customization: KServe is highly customizable, allowing you to define how you want to serve your models, handle preprocessing and post-processing tasks, and even add custom logic to your serving containers.
+6. Integration with Kube Flow Pipelines: If you're using Kube Flow Pipelines for end-to-end machine learning workflows, KServe can be integrated seamlessly to deploy models as part of your pipeline.
+
+Visit the [Kserve documentation](https://kserve.github.io/website/0.10/modelserving/control_plane/) for more information.
+
+The first step of model deployment is to define the service account with MinIO object store secrets.
+
+![](/img/fig-1a-define-service-acct.png "Figure 8. Defining service account with MinIO secret. ")
+
+Next, script the KServe yaml file to begin the inference service. 
+
+![](/img/fig-1b-inference-service.png "Figure 9. The inference service is defined. ")
+
+Next, a new model server must be created in Kube Flow and the yaml file uploaded. The inference service is created and once it is running, we can send our test data through REST APIs and get the response.
+
+**Model Retraining**
+
+The models deployed into production are constantly monitored to identify inconsistencies and degradation in performance. Because models can skew to the old data used for training resulting in under performance, models should be retrained whenever performance drops below a defined threshold. To retrain a model, a pipeline is developed that could trigger all the steps associated with model retraining to occur in the appropriate sequence using Kube Flow pipelines.  
```

Comment:
In this section, please check occurrences of "Kube Flow": Is it Kube Flow or Kubeflow?